services/apps/alcs/src/application/application-document/application-document.service.ts

```diff
@@ -100,4 +102,25 @@ export class ApplicationDocumentService {
   async getDownloadUrl(document: ApplicationDocument) {
     return this.documentService.getDownloadUrl(document.document);
   }
+
+  async attachExternalDocuments(
+    fileNumber: string,
+    data: ApplicationDocumentCreateDto[],
+  ) {
+    const application = await this.applicationService.getOrFail(fileNumber);
+
+    const documentsToSave: ApplicationDocument[] = [];
```

Comment:
nitpick: I think it would be cleaner using a .map instead of creating a new array and using an of loop to push